src/persistentmodel/job.cpp

```diff
@@ -57,8 +58,10 @@ void Job::setUrls(const QList<QUrl> &urls)
 
 bool Job::validateUrls()
 {
+    // Check that _urls is not empty.
```

Comment:
Do we really need overly explicit comments like these? They are superfluous if you ask me.